recipe/meta.yaml

```diff
@@ -11,21 +14,80 @@ source:
     - 0001-Fix-build-on-ppc.patch  #[ppc64le] 
 
 build:
-  number: 1
-  string: py{{ python | replace(".", "") }}_{{ PKG_BUILDNUM }}
-  
-requirements:
-  build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - bazel {{ bazel }}
-    - git >={{ git }}
-  host:
-    - python  {{ python }}
-    - tensorflow-base {{ tensorflow }}
-  run:
-    - python  {{ python }}
-    - tensorflow-base {{ tensorflow }}
+  number: 2
+  run_exports:
+    - {{ pin_subpackage("tensorflow-addons", max_pin="x.x.x") }}
+
+outputs:
+  - name: tensorflow-addons-proc
+    version: {{ build_ext_version }}
+    build:
+      number: {{ proc_build_number }}
+      string: {{ build_ext }}
+    test:
+      commands:
+        - exit 0
+    about:
+      home: https://www.tensorflow.org/addons/
+      license: Apache-2.0
+      license_family: Apache
+      license_file: {{ SRC_DIR }}/LICENSE
+      summary: "A meta-package to select TensorFlow addons build variant"
+
+  - name: tensorflow-addons
+    script: build-addons.sh
+    version: {{ version }}
+    build:
+      string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}{{ cudatoolkit | replace(".*", "") }} #[build_type == 'cuda']
+      string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ build_ext }}  #[build_type == 'cpu']
+{% if build_type == 'cuda' %}
+      script_env:
+        - CUDA_HOME
+{% endif %}
+      run_exports:
+        - {{ pin_subpackage("tensorflow-addons", max_pin="x.x.x") }}
+      track_features:
+        {{ "- tensorflow-addons-cuda" if build_type == 'cuda' else "" }} 
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - bazel {{ bazel }}
+        - git >={{ git }}
+      host:
+        - python  {{ python }}
+{% if build_type == 'cuda' %}
+        - cudatoolkit {{ cudatoolkit }}
+        - cudnn {{ cudnn }}
+        - nccl {{ nccl }}
+{% endif %}
+        - tensorflow-base {{ tensorflow }}
```

Comment:
With that I get:
```
 Unsatisfiable dependencies for platform linux-ppc64le: {'tensorflow-base=2.4[build=*cuda]'}
```
So that new notation is kind of incompatible with the older meta-package method.
So I just changed it to:
```
        - tensorflow {{ tensorflow }}  #[build_type == 'cuda']
        - tensorflow-cpu {{ tensorflow }} #[build_type == 'cpu']

```